common/source/docs/common-dock-mode.rst

```diff
@@ -0,0 +1,103 @@
+
+- Connect the OpenMV camera to your PC using  a micro USB cable.
+- Open the OpenMV IDE and click the connect button at the bottom left corner of the IDE window.
+- Go to file menu, create new file and copy `this <https://github.com/shiv-tyagi/openmv-examples/blob/master/apriltag_autodock.py>`__ script to the editor window.
+- Read instructions about setting the marker size and id at the beginning of the script (after the LISENCE and copyright notice). The marker detection range can be improved by using large sized markers.
+- To upload the code to the camera go to Tools menu and click on `Save open script to OpenMV Cam (as main.py)`.
+- Done! Now you can remove the microUSB cable and power up the camera using an external power source.
+  You should see a RED LED (or GREEN LED if the target is in sight of the camera) blinking on the camera when the script is running.
+
+Mounting to the vehicle
+=======================
+
+The camera should be mounted at the front of the vehicle. The camera board should be mounted such that the camera edge with
+the lens is upwards (i.e., away from the ground). This is the default setup configuration that we think would be used
+in most of the cases. However, support for mounting the camera at the back and in different mounting orientations is also provided.
+If you want to mount the camera in any other configuration, please don't forget to set the related `PLND_XXX` parameters accordingly.
+
+.. note::
+    The precision landing library also relies on the vehicle's compass to construct the target vector.
+    Any kind of `magnetic interference <https://ardupilot.org/rover/docs/common-magnetic-interference.html>`__ in the vehicle's compass may lead to the vehicle behaving inappropriately during the docking process.
+
+Setup through GCS
+=================
+
+The following parameters are needed to be set to enable the precision landing using a companion computer on the rover.
+
+-  :ref:`PLND_ENABLED <PLND_ENABLED>` = 1
+-  :ref:`PLND_TYPE <PLND_TYPE>` = 1
+You would need to reboot the vehicle once you set up the above mentioned parameters.
+
+If you are mounting the camera in any configuration other than the default configuration mentioned in the previous section, please set the following parameters appropriately.
+
+-  :ref:`PLND_CAM_POS_X <PLND_CAM_POS_X>`, :ref:`PLND_CAM_POS_Y <PLND_CAM_POS_Y>`, :ref:`PLND_CAM_POS_Z <PLND_CAM_POS_Z>` : Position of camera in body frame (in m)
+-  :ref:`PLND_ORIENT <PLND_ORIENT>` : Orientation of camera on vehicles body. The camera can be mounted at the front or at the back of the vehicle's body
+-  :ref:`PLND_YAW_ALIGN <PLND_YAW_ALIGN>` : Angle of rotation of camera from default configuration to mounted configuration around the axis pointing out from it's lens (in centi degrees)
+
+The docking process can be fine tuned using the following parameters.
+
+- :ref:`DOCK_SPEED <DOCK_SPEED>` : The maximum speed limit at the which the vehicle can maneuver in the dock mode.
```

Comment:
any ref link must have no whitespace after it and before a comma,period, or colon...lines 94-98 are generating these build errors...